kube/src/lib.rs

```diff
@@ -431,6 +434,103 @@ mod test {
         Ok(())
     }
 
+    #[tokio::test]
+    // #[ignore] // needs cluster (fetches api resources, and lists all)
```

Comment:
needs to be ignored again, fails unit test job

Reply:
done.